tests/Microsoft.Bot.Connector.Tests/GetTokenRefreshTests.cs

```diff
@@ -25,6 +25,14 @@ public async Task TokenTests_GetCredentialsWorks()
             Assert.NotNull(result);
         }
 
+        [Fact]
+        public async Task TokenTests_GetCredentialsFromTenantWorks()
+        {
+            MicrosoftAppCredentials credentials = new MicrosoftAppCredentials("645cd89f-a83e-4af9-abb5-a454e917cbc4", "jvoMWRBA67:zjgePZ359_-_", "microsoft.com", null);
```

Comment:
2 things here:
1. We cannot put magic keys / constants in code. If we did this, the keys needs to be stored as encrypted variables in Azure Devops and passed into the build via ENV variables. (This is a super-common pattern). 

2. We cannot put actual "Service" tests here. These are unit tests, and cannot hit actual services.

Reply:
this is how tests are written in this file. I'm copying the existing pattern.

Reply:
Alright, I'll clean this up seperatly.